<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>611</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QRadioButton" name="rB_annuity">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Annuity</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_differentiated">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Differentiated</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>298</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="l_credit_amount">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Credit Amount</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="dSB_credit_amount">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_interest_rate">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Interest Rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="dSB_interest_rate">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_credit_term">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Credit Term</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sB_credit_term">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>298</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="l_interest_charges">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Interest Charges</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="dSB_interest_charges">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_monthly_payments">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Monthly Payments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="dSB_monthly_payments">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="l_total_payout">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Payout</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="dSB_total_payout">
      <property name="font">
       <font>
        <family>Fira Code</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pB_calculate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
